Sources/SpeziLicense/Views/ContributionsList.swift

```diff
@@ -0,0 +1,64 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftPackageList
+import SwiftUI
+
+
+public struct ContributionsList: View {
+    var packages: [Package]
+    
+    
+    public var body: some View {
+            List {
+                Section(footer: Text("This project is licensed under the MIT License.", bundle: .module)) {
+                    Text("The following list contains all Swift Package dependencies of the SpeziTemplateApplication.", bundle: .module)
+                }
```

Comment:
As noted in the Template Application PRs we will need a way to configure the license of the project using this list as well as the name of the project and should be able to pass it in as arguments to this SwiftUI View.